docs/howto/content/python.md

```diff
@@ -39,3 +39,41 @@ os.getcwd()  # If successful:  "/drive/path/to/notebook"
   https://github.com/jupyterlite/jupyterlite/blob/main/examples/pyolite/emscripten-filesystem.ipynb
 [fs]: https://emscripten.org/docs/api_reference/Filesystem-API.html
 [caniuse-sw]: https://caniuse.com/serviceworkers
+
+## Fetching remote content
+
+It is also possible to fetch content from remote URL. For example if you are using the
+Pyodide kernel, you can use the `fetch` function to do so:
+
+```py
```

Comment:
Probably add headers here, and have a link to `pyodide.http`, as `py_fetch` is also quite usable:

https://pyodide.org/en/stable/usage/api/python-api/http.html